install-dev/data/db_structure.sql

```diff
@@ -829,7 +829,8 @@ CREATE TABLE `PREFIX_employee` (
   PRIMARY KEY (`id_employee`),
   KEY `employee_login` (`email`, `passwd`),
   KEY `id_employee_passwd` (`id_employee`, `passwd`),
-  KEY `id_profile` (`id_profile`)
+  KEY `id_profile` (`id_profile`),
+  KEY `IDX_1D8DF9EBBA299860` (`id_lang`)
```

Comment:
do you know why this index is removed ?

Reply:
It's not removed, it's added The initial table creation only created one And Doctrine systematically added this extra one via the command